models/sla_policy/reply_time/int_zendesk__reply_time_business_hours.sql

```diff
@@ -62,7 +62,8 @@ with ticket_schedules as (
 
     from ticket_sla_applied_with_schedules
     cross join weeks
-    where {{ fivetran_utils.ceiling('target/total_schedule_weekly_business_minutes') }} >= generated_number - 1
+    -- We need to understand more why the below is needed. I feel this is more a performance issue than a data quality issue. Still need to confirm.
```

Comment:
Did we want to remove the clause still? I do agree this was here to make the cross join more performant

Reply:
Ah yeah thanks for catching, this is not needed anymore. Will remove

Reply:
Removed.